test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -79,7 +79,7 @@
     ("aten::mkldnn_linear", datetime.date(2021, 3, 2)),
     ("aten::_mode*", datetime.date(2021, 5, 2)),
     ("aten::linalg_multi_dot", datetime.date(2021, 3, 25)),
-    ("aten::coalesce", datetime.date(2021, 3, 31)),
```

Comment:
landing takes such long time? :-)

Reply:
we gotta make backwards compatibility tests more resilient :(